### ISSUE TITLE
Ability to monitor a file for changes

### ISSUE THREAD

**Issue by gfwilliams** [espruino/espruino-tools#8]
Like this? https://github.com/olliephillips/espruingo

Looks like a great idea - so you can easily use your own editor.

**Comment by loop23** [comment]
Yes. Just throw in minification, module loading and maybe compilation and it would be perfect. I miss emacs on the web IDE!

**Comment by gfwilliams** [comment]
Yes - so all that can happen easily with EspruinoTools (it's the whole WebIDE pipeline).
